### DISCUSSION TITLE
erratic snapping in v9 when swiping diagonally

### DISCUSSION THREAD

**Discussion by aus70** [nolimits4web/swiper#6376] [General]
Hi

I upgraded to v9 and - at least on iOS devices - the snapping seems to break when I pull the page down and left, which is something that happens often when swiping with the right-hand thumb. When I raise the thumb, the slides do not snap into position, but 2 slides remain visible. v8 somehow does not allow to swipe diagonally, but only vertically or horizontally, depending on the initial swipe direction.  Does anybody else have the same issue?

Thanks.

**Comment by eeschiavo** [comment]
Same issue here
